lib/vicadmin/validate.go

```diff
@@ -186,7 +187,7 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 	nwErrors := []error{}
 
 	// create a http client with a custom transport using the proxy from env vars
-	client := &http.Client{Timeout: 30 * time.Second}
+	client := &http.Client{Timeout: 10 * time.Second}
```

Comment:
So `10 sec` or `5 sec`? I thought you wanted to reduce the timeout to 5 sec

Reply:
Talked with @hmahmood and settled on 10, @chengwang86

Reply:
Sorry, didn't update the original description after the first commit ;P